.github/workflows/ruby.yml

```diff
@@ -56,3 +56,8 @@ jobs:
       - name: Run tests
         working-directory: demo
         run: bundle exec rake test:all
+      - name: Commit changed screenshots and other files
+        if: failure()
+        working-directory: demo
+        run: bundle exec rake commit
+
```

Comment:
This will automatically commit all changed files in the demo directory after a failure?  That does not seem right.

Reply:
I couldn't think of a better way to communicate the changed screenshots back into a place where the dev could review the screenshots and see if their code needed to be fixed. GitHub's diff for the screenshots is actually pretty slick. I admit that the revert step may be a nuisance, or people may forget it. But we should still catch changes when reviewing the PR.

I could make the workflow more specific, so we _only_ save the screenshot changes when it was the screenshot diff that failed.

I'm open for other suggestions about how to generate and capture the screenshots when we create new ones, or the change is intended.